web-wallet/src/lib/components/Transactions/Transactions.svelte

```diff
@@ -91,20 +96,51 @@
 								data-tooltip-place="top"
 							/>
 						</dd>
-					{/if}
-				</dl>
-			{/each}
-		{:else}
-			<div class="transactions-list__empty">
-				<Icon path={mdiContain} size="large"/>
-				<p>You have no transaction history</p>
-			</div>
-		{/if}
-	</div>
+						{#if transaction.direction === "Out"}
+							<dt class="transactions-list__term">Fee</dt>
+							<dd class="transactions-list__datum">
+								{feeFormatter(transaction.fee)}
+								<Icon
+									className="transactions-list__icon"
+									path={logo}
+									data-tooltip-id="transactions-tooltip"
```

Comment:
As you are refactoring here: the component is in the application domain, so there is no reason to have an extra tooltip element (and its event handlers and observer). Just use the `main-tooltip`.

Reply:
I do agree, refactored.